src/scripts/admin/filters/button-enhanced.js

```diff
@@ -110,8 +108,7 @@ const addInspectorControl = createHigherOrderComponent((BlockEdit) => {
 									setAttributes({ svgIcon: value });
 								}}
 							/>
-						)}
-						{!clickFlag && (
+						) : (
 							<>
 								<h3>Instructions (for CleanBC only):</h3>
```

Comment:
you can import https://www.npmjs.com/package/@wordpress/i18n for javascript i18n